lib/internal/Magento/Framework/Image/Adapter/Gd2.php

```diff
@@ -778,4 +778,68 @@ protected function _createEmptyImage($width, $height)
         $this->imageDestroy();
         $this->_imageHandler = $image;
     }
+
+    /**
+     * Fix an issue with the usage of imagecopymerge where the alpha channel is lost
+     *
+     * @param resource $dst_im
+     * @param resource $src_im
+     * @param int $dst_x
+     * @param int $dst_y
+     * @param int $src_x
+     * @param int $src_y
+     * @param int $src_w
+     * @param int $src_h
+     * @param int $pct
+     *
+     * @return bool
+     */
+    private function imagecopymergeWithAlphaFix(
```

Comment:
Please just mention in PHPDoc of this method that signature matches http://php.net/manual/en/function.imagecopymerge.php function but name arguments properly: `$destinationImage`, `$sourceImage`, `$destinationX`, etc.